committee-steering/governance/working-group-formation-process.md

```diff
@@ -0,0 +1,87 @@
+# Kubernetes Working Group Formation Process - Proposal
+
+Draft 1.0 // Jaice Singer DuMars // June, 2018
+
+## Process Overview and Motivations
+As opposed to “Birds of a Feather” or discussion groups, Working Groups provide a more formal avenue for disparate
+groups to intellectually collaborate around a common problem, craft a balanced position, and disband. Because they
```

Comment:
delete "intellectually" as it is semi vague and isn't necessary to the intent of the sentence.

Reply:
Done